specification/common-types/resource-management/v4/privatelinks.json

```diff
@@ -28,7 +28,7 @@
       },
       "allOf": [
         {
-          "$ref": "../v3/types.json#/definitions/Resource"
+          "$ref": "../v4/types.json#/definitions/Resource"
```

Comment:
>v4 [](http://example.com/codeflow?start=22&length=2)

Would this not break existing implementations that are based on v3?

Reply:
I don't think this breaks anyone since the only diff is a new `x-ms-format` which direct consumers of openapi wouldn't be looking for and all internal consumers always fall back to `format` if they don't understand what is inside of `x-ms-format`.

Reply:
The only RP that references this type is https://github.com/Azure/azure-rest-api-specs/blob/main/specification/iotcentral/resource-manager/Microsoft.IoTCentral/preview/2021-11-01-preview/iotcentral.json#L353 which is in `preview`. Their stable version [021-06-01](https://github.com/Azure/azure-rest-api-specs/blob/main/specification/iotcentral/resource-manager/Microsoft.IoTCentral/stable/2021-06-01/iotcentral.json) doesn't use it